src/python/visclaw/plotpages.py

```diff
@@ -970,9 +982,12 @@ def plotclaw2kml(plotdata):
 
     try:
         setgauges = gaugetools.read_setgauges(plotdata.outdir)
+        have_gauges = True
     except:
         print("     File gauges.data not found.")
-    else:
+        have_gauges = False
```

Comment:
Don't you want to try to read the data and then check the file to see if you have gauges or not?  If you cannot read the gauge.data file at all you cannot do anything below regardless.